vue/components/ui/molecules/form-input/form-input.vue

```diff
@@ -53,6 +48,12 @@
     flex: 1;
 }
 
+.form-input .header.header-underline {

```

Comment:
No need for this new `header-underlin` class. When `hasTooltip` is `tue`, the tooltip will be wrapping the header thus, you can do `.form-input .tooltip > .header`. Also remove that class `header-underline` from the computed `headerClasses` bellow

Reply:
You're right, this was a remnant of a previous version that remained, thanks!